components/brave_wallet/browser/tx_service.cc

```diff
@@ -277,6 +278,13 @@ void TxService::Reset() {
   }
 }
 
+void TxService::MakeFilForwarderTransferData(
+    const std::string& to_address,
+    MakeFilForwarderTransferDataCallback callback) {
+  auto bytes = FilAddress::FromAddress(to_address).GetBytes();
+  GetEthTxManager()->MakeFilForwarderTransferData(bytes, std::move(callback));
```

Comment:
Let's make FilAddress from address string, check that it is a valid address and pass it to TxManager as is(no conversion to bytes).